src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -56,91 +54,109 @@ const defaultProps = {
     loginList: {},
 };
 
+const getPhoneLogin = (phoneOrEmail) => {
+    if (_.isEmpty(phoneOrEmail)) {
+        return '';
+    }
+
+    return LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(phoneOrEmail));
+};
+
+const validateNumber = (values) => {
+    const parsedPhoneNumber = parsePhoneNumber(values);
+
+    if (parsedPhoneNumber.possible) {
+        return parsedPhoneNumber.number.e164;
+    }
+
+    return '';
+};
+
 function NewContactMethodPage(props) {
-    const [login, setLogin] = useState('');
-    const [password, setPassword] = useState('');
     const loginInputRef = useRef(null);
 
-    const handleLoginChange = useCallback((value) => {
-        setLogin(value.trim());
-    }, []);
-
-    const handlePasswordChange = useCallback((value) => {
-        setPassword(value.trim());
-    }, []);
+    const isFormValid = (values) => {
```

Comment:
We can rename this because _is_ expecting the return to be boolean, like what the method does before. For reference, most of the other component use `Form.js` naming the validation callback function is `validate`.